webpack/webpack.dev.babel.js

```diff
@@ -29,13 +31,20 @@ let REPLACEMENT_VARS = {
   ENV: environment
 };
 
-let entry = [
-  `webpack-dev-server/client?http://localhost:${PORT}`,
-  './src/js/index.js'
-];
+let dependencies = Object.assign({}, packageInfo.dependencies);
+delete dependencies['canvas-ui'];
+delete dependencies['cnvs'];
+
+let entry = {
+  index: ['./src/js/index.js'],
+  vendor: Object.keys(dependencies)
```

Comment:
We should take care to understand how the plugin splits out these vendor files. My guess is it uses regex to match dependency names against file paths. If that is the case then we could risk splitting out some non-vendor code into the vendor bundle if we have folders or files that contain a vendor name.